clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/ops/AbstractEnableDisableAtomicOperation.groovy

```diff
@@ -129,7 +130,12 @@ abstract class AbstractEnableDisableAtomicOperation implements AtomicOperation<V
 
             describeInstancesRequest.setNextToken(describeInstancesResult.nextToken)
           } catch(Exception e) {
```

Comment:
The other thing would be doing a DescribeInstances w/ a filter of the asg being xyz ... vs fetching the ASG and doing `.instances`